etc/clang-flags.cmake

```diff
@@ -3,7 +3,7 @@ include_guard(GLOBAL)
 set(CMAKE_CXX_STANDARD 23)
```

Comment:
remind me -- is there something here that requires 23, or could it be 20?

Reply:
I'll check. I believe at one point I was looking at `deducing this` but then didn't.